mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a,b,c) {
   return a + b + c;
 }
```

Comment:
That is correct, when defining parameters in a function, you should always separate the parameters with a `,`